src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -87,6 +88,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $this->convertController($route);
 
+            // Converts a short notation service:method to a class::method.
+            if ($route->hasDefault('_controller') && 'txt' === $input->getOption('format')) {
+                list($service, $method) = explode(':', $route->getDefault('_controller'), 2);
+                try {
+                    $route->setDefault('_controller', sprintf("%s:%s\n   (service: %s)", get_class($this->getContainer()->get($service)), $method, $service));
```

Comment:
should be `::` for fully-qualified class name and method ([CNP](http://symfony.com/doc/current/routing.html#controller-naming-pattern))